apps/landingPage/src/components/Headline.js

```diff
@@ -75,137 +75,139 @@ const HeadlineContainer = styled.div`
-  width: 100vw;
-  background-image: linear-gradient(
-    #e9eef1,
-    50%,
-    ${defaultTokens.paletteWhite}
-  );
-`;
-
 const HeaderAndDesignWrapper = styled.div`
-  width: 50vw;
-  padding-left: 10vw;
-  @media (max-width: ${BREAKPOINTS.TABLET}px) {
-    padding-left: 0;
-    width: 90vw;
-  }
-`;
-
-const BackgroundGradientAccent = styled.div`
-  position: absolute;
-  z-index: -1;
-  display: flex;
-  align-items: center;
-  height: 70vh;
-  width: 40vw;
-  margin: -8vh 0 0 -11vw;
-  background-image: linear-gradient(
-    #e9eef1,
-    50%,
-    ${defaultTokens.paletteWhite}
-  );
-
-  @media (max-width: ${BREAKPOINTS.TABLET}px) {
-    display: none;
+  padding-left: 0;
+  width: 90vw;
+  @media (min-width: ${BREAKPOINTS.TABLET}px) {
+    width: 50vw;
+    padding-left: 10vw;
   }
 `;
 
 const GreenRectangle = styled.div`
+  display: none;
   height: 167px;
-  width: 3vh;
+  width: 30px;
   background-color: ${defaultTokens.paletteProductNormal};
-
-  @media (max-width: ${BREAKPOINTS.TABLET}px) {
-    display: none;
+  @media (min-width: ${BREAKPOINTS.TABLET}px) {
+    display: block;
   }
 `;
 
 const ContainerIcons = styled.div`
   display: flex;
   justify-content: space-around;
-  width: 28vw;
+  width: 100%;
   margin-right: 3vw;
-  @media (max-width: ${BREAKPOINTS.TABLET}px) {
-    width: 80%;
+  @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) and (
+    max-width: ${BREAKPOINTS.TABLET}px) {
+    width: 60%;
     margin: 40px 0 40px 0;
     justify-content: space-evenly:
   }
-  @media (max-width: ${BREAKPOINTS.BIG_MOBILE}px) {
-    width: 100%;
+  @media  (min-width: ${BREAKPOINTS.TABLET}px) {
+    width: 28vw;
   }
-
 `;
 
 const ContainerBottom = styled.div`
   display: flex;
-  justify-content: space-between;
-  padding: 9.5vh 0 0 2vw;
-  @media (max-width: ${BREAKPOINTS.TABLET}px) {
-    flex-direction: column;
-    padding: 0;
-    align-items: center;
+  flex-direction: column;
+  padding: 0;
+  align-items: center;
+  @media (min-width: ${BREAKPOINTS.TABLET}px) {
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 9.5vh 0 0 2vw;
+    align-items: flex-start;
   }
 `;
 
-const PlaneImage = styled.div`
-  width: 57vw;
-  height: 50vh;
+const MobileGradientAccent = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  height: 300px;
+  width: 100vw;
+  background-image: linear-gradient(
+    #e9eef1,
+    50%,
+    ${defaultTokens.paletteWhite}
+  );
+`;
+
+const BackgroundGradientAccent = styled(MobileGradientAccent)`
+  align-items: center;
+  justify-content: flex-start;
+  height: 70vh;
+  width: 40vw;
+  position: absolute;
+  z-index: -1;
+  margin: -8vh 0 0 -11vw;
+  display: none;
+  @media (min-width: ${BREAKPOINTS.TABLET}px) {
+    display: flex;
+  }
+`;
+
+const PlaneImageMobile = styled.div`
+  width: 94vw;
+  height: 280px;
   background-image: url(${airport});
   background-size: cover;
   background-repeat: no-repeat;
   background-position: 50% 50%;
```

Comment:
Just a minor thing. But maybe we can set this to something around `50% 90%` to have plane in focus in some edge cases.

`50% 50%`
<img width="743" alt="screenshot 2019-02-27 at 10 43 34" src="https://user-images.githubusercontent.com/2660330/53481067-bc3d4b00-3a7c-11e9-8f98-5837e2167692.png">

`50% 90%`
<img width="743" alt="screenshot 2019-02-27 at 10 43 47" src="https://user-images.githubusercontent.com/2660330/53481075-bf383b80-3a7c-11e9-80c3-fbc53a572375.png">

Reply:
Yes, looks much better like that.